codice/ddf
# DDF-4401 Adding System Templates Karaf Command

#### What does this PR do?
Adds Karaf command to manage system templates.
#### Who is reviewing it? 
@Bdthomson 
@GabrielFabian 
@middlets719 
@zta6 
@mojogitoverhere 

#### Select relevant component teams: 

#### Ask 2 committers to review/merge the PR and tag them here.

@bdeining 
@millerw8 
@djblue 

#### How should this be tested?
Run the command `forms:load` to invoke the ingestion of the system templates. Ensure that system templates are placed in the proper directory or else nothing will be ingested.

Run `forms:manage -l / forms:manage --list` to view all ingested forms.
Run `forms:manage -s / forms:manage --remove-single "ID"` to remove a single metacard.
Run `forms:manage -r / forms:manage --remove-all` to remove all system templates.

#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-4401](https://codice.atlassian.net/browse/DDF-4401)
#### Screenshots

#### Checklist:
- [ x ] Update / Add Unit Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
